planner/core/rule_join_reorder_greedy.go

```diff
@@ -110,7 +127,7 @@ func (s *joinReorderGreedySingleGroupSolver) constructConnectedJoinTree() (Logic
 			if err != nil {
 				return nil, err
 			}
-			curCost := curJoinTree.statsInfo().RowCount + newJoin.statsInfo().RowCount + node.statsInfo().RowCount
+			curCost := s.nodeCost(curJoinTree) + newJoin.statsInfo().RowCount + s.nodeCost(node)
```

Comment:
```suggestion
			curCost := s.nodeCost(curJoinTree) + s.nodeCost(newJoin) + s.nodeCost(node)
```
should be this?

Reply:
another thing is, should we calculate the cumulative cost of `newJoin` instead of directly using `newJoin.statsInfo().RowCount`?

Reply:
Emm... Actually, the calculated `curCost` is the cumulative cost of `newJoin`.